addons/explosives/scripts/TripflareEffect.sqf

```diff
@@ -0,0 +1,20 @@
+/*
+ * Author: VKing
+ * Spawns a flare on the ground for tripflare trigger
+ *
+ * Arguments:
+ * 0: Position <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * beforeDestroyScript = "\z\ace\addons\explosives\scripts\TripflareEffect.sqf;
+ *
+ * Public: no
+ */
+
+//This needs to be a script file, so it will be re-compiled each use.
```

Comment:
Is there a reason it has to be recompiled on each use? Add the reason to comments maybe.